fluster/decoders/av1_aom.py

```diff
@@ -41,9 +41,14 @@ def decode(
         keep_files: bool,
     ) -> str:
         """Decodes input_filepath in output_filepath"""
+        fmt = None
+        if output_format == OutputFormat.YUV420P:
+            fmt = "--i420"
```

Comment:
So for 420, it won't produce I420 with --rawvideo ? Just wondering, since if its NV12, it would not be surprising if --rawvideo propduces P010/P012 instead of I420 10bit.

Reply:
I checked with yuview, --rawvideo produces I420 10 bit yeah.

Reply:
I meant to double heck we need the exception and can't use --rawvideo for I420 (8bit) too